climpred/utils.py

```diff
@@ -97,3 +103,52 @@ def reduce_time_series(forecast, reference, nlags):
     forecast = forecast.where(forecast.time >= imin, drop=True)
     reference = reference.where(reference.time >= imin, drop=True)
     return forecast, reference
+
+
+def assign_climpred_compute_to_attrs(
```

Comment:
This could just be trimmed down to `assign_attrs()` unless you see additional functions being added outside of `compute` attrs. So it could also be `assign_compute_attrs()`.

Reply:
I like assign_attrs to be consistent with https://xarray.pydata.org/en/stable/generated/xarray.Dataset.assign_attrs.html

Reply:
change to `assign_attrs()`